wpimath/src/main/java/edu/wpi/first/math/filter/SlewRateLimiter.java

```diff
@@ -14,38 +14,39 @@
  * edu.wpi.first.math.trajectory.TrapezoidProfile} instead.
  */
 public class SlewRateLimiter {
-    private final double m_rateLimit;
     private double m_prevVal;
     private double m_prevTime;
     private double m_forwardRateLimit;
     private double m_backwardRateLimit;
 
     /**
      * Creates a new SlewRateLimiter with the given rate limit and initial value.
+     * Sets the given rate limit to both the forward and backward limits.
      *
      * @param rateLimit    The rate-of-change limit, in units per second.
      * @param initialValue The initial value of the input.
      */
     public SlewRateLimiter(double rateLimit, double initialValue) {
```

Comment:
I'd remove this ctor; users can immediately call `reset` if needed.  We shouldn't have two different ctors with (double, double) signatures.